gateway/src/apicast/cli/filesystem.lua

```diff
@@ -9,9 +9,9 @@ local pl_path_dir = pl_path.dir
 local pl_path_join = pl_path.join
 local abspath = pl_path.abspath
 local pcall = pcall
-local co_yield = coroutine.yield
-local co_create = coroutine.create
-local co_resume = coroutine.resume
+local co_yield = coroutine._yield
```

Comment:
I think it would be good to add a comment that clarifies why we're using the original methods here.

Reply:
Yeah, I'll write a comment.

Reply:
In the end, this is not necessary. It would have to be fixed in the penlight library anyway.